recipes/multi-hauser/README.md

```diff
@@ -0,0 +1,72 @@
+# Multi-Hauser with Docker
+
+If your FullStory account has multiple associated accounts, this recipe can be used as a template to
+start a docker container that exports the data for each "org" and uploads it to BigQuery.
+While this is intended to work with Google Cloud, it can easily be adapted to work with
+AWS or any databases supported in the future.
+
+## How it works
+This recipe works by creating a docker container that uses [supervisord] to run multiple `hauser`
+processes against multiple FullStory accounts (i.e. "orgs").
+When the docker container is started, it uses the provided [key file](#key-file) to build a `config.toml` for each org from the
+[hauser config template](hauser-config.toml.tmpl) and saves it to a working directory for that org
+(e.g. `/hauser/<orgId>/config.toml`).
+A ["program"](http://supervisord.org/configuration.html#program-x-section-settings) is also added to the supervisor config
+file.
+Supervisor is then started and runs in the foreground until the container is terminated.
+
+## Hauser Configuration Template
+The template in this directory requires that three variables are defined:
+ * `Bucket`: Google Cloud Storage bucket ([gcs] section)
+ * `Project`: Google Cloud project ID ([bigquery] section)
+ * `Dataset`: BigQuery dataset that the tables will be created in ([bigquery] section)
+
+The other `hauser` configuration variables can be customized as well, but the above are the minimum required.
+
+#### Export Delay
+In an attempt to offset the times at which each process is doing work,
+the generated `hauser` configs will have a 10 min offset in their `ExportDelay` value.
+So, if your [key file][key file] has 3 lines, the `ExportDelay` value for each org will be
+24 hours, 24 hours and 10 minutes, and 24 hours and 20 minutes, respectively.
+
+## Key File
+The key file should be a CSV with the following format:
+
+```
+<org1>,<api_key_for_org1>
+<org2>,<api_key_for_org2>
+<org3>,<api_key_for_org3>
+...
```

Comment:
I think the CSV file must have a new line at the end or it'll drop the last line.  (I had tried this approach with a single line with org:apikey pair, and it didn't work).  So perhaps a tip or nota bene related to this requirement could be added.

Reply:
Ooh, good call. My editors add that automatically so I didn't realize I needed to call it out.